SpeechTranslator/CS/README.md

```diff
@@ -0,0 +1,85 @@
+  - Windows 10 IoT Enterprise
+description: Translate your speech to different languages.
+---
+
+# Build the Speech Translator Project
+
+## Component Lists
+- A Windows 10 IoT Enterprise Device (e.g. UP Board)
+- A headset [e.g. the Microsoft LifeChat-3000 Headset](http://www.microsoft.com/hardware/en-us/p/lifechat-lx-3000/JUG-00013) 
+- A mouse
+- A router connected to the Internet 
+- An ethernet cable
+- An HDMI monitor and cables 
+- A micro-SD card and reader
+
+## Step 1: Setup your Development PC (Required for Windows IoT Builds 15063 or greater.)
+1. Install the Windows ADK for Windows 10, version from [here](https://developer.microsoft.com/en-us/windows/hardware/windows-assessment-deployment-kit)
+    - Ensure that "Imaging And Configuration Designer" (ICD) is selected.
+    - Install to the default location.    
+2. Install the Windows IoT Enterprise ADK Add-Ons from [here](https://developer.microsoft.com/en-us/windows/hardware/windows-assessment-deployment-kit)
+    - Install to the default location.
+  
+## Step 2: Setup your IoT Device
+1. Install [Windows 10 IoT Enterprise](https://docs.microsoft.com/windows/iot/iot-enterprise/downloads) to your device 
+2. Connect your device to the router, and connect the router to the Internet.
+3. Connect the headset and mouse to your device.
+4. Boot your device.
+5. Select a language and, if Wi-Fi is supported on your IoT Device, skip the Wi-Fi Configuration step.
+6. Provide Voice Permission from Cortana when prompted.  **(Required for Windows 10 IoT Builds 15063 or greater)**
+    1. At the first Cortana Prompt, **press OK**.
+    2. At the second Cortana Prompt, press **Maybe Later**.     
+7. Rename your device
+    1. Use Powershell to connect to your device
+    2. Change the name of your device to "speechtranslator"
+        - **setcomputername speechtranslator**
+    3. Reboot your device.
+8. Copy a Speech Language Package to your device.  **(Required for Windows 10 IoT Builds 15063 or greater)**
+    1. Open a new powershell window on your desktop
+    2. From desktop map your device's disk to a local drive:
+        - **net use t: \\speechtranslator\c$ /user:\administrator p@ssw0rd)**
+    3. Copy a language package from your desktop to your device.  
+        *Note: Language packages are available in this folder:* c:\Program Files (x86)\Windows Kits\10\MSPackages\retail\ **your-processor-type** \fre
+        
+        For example, to copy Spanish Mexican to an arm based device:
+        - **copy "c:\Program Files (x86)\Windows Kits\10\MSPackages\retail\amd64\fre\Microsoft-Windows-OneCore-Microsoft-SpeechData-es-MX-Package.cab" t:**                   
+    4. Unmap your local drive (e.g. net use /delete t:).
+    
+9. Apply the Speech Language Package to your device.  **(Required for Windows 10 IoT Builds 15063 or greater)**
+    
+    *Note: Your device should reboot after committing each update and a "spinning gears" screen will appear until the update completes.*

```

Comment:
Does the spinning gears show up for iot enterprise?

Reply:
Just Reboot happens, Removed the Note